### ISSUE TITLE
Hint for the password field

### ISSUE THREAD

**Issue by boryn** [rtyley/agit#39]
I don't have yet SSH keys configured and I enter the password in the field. 

I'd suggest giving the field a hint (I mean greyed out text, like for e.g. clone_directory_hint)

And second - the password should not be visible in the field! Or there could be a checkbox "show password" and when the user enables the option, the password shows up, otherwise it is not visible (ony the letter we currently enter)

**Comment by rtyley** [comment]
Hi @boryn - I'm assuming you mean the 'pop-up' password prompt that occurs when you're accessing a Git repo via SSH using password-authentication.

I'm doing a bit of work on the Agit/SSH interaction at the moment, I'll definitely look into giving the field a hint (e.g. 'Password') and also making it hide the password characters.

**Comment by boryn** [comment]
Yes, yes - it's exactly the 'pop-up' password prompt (didn't know how to express it properly :))
